zcash_client_sqlite/src/wallet.rs

```diff
@@ -524,21 +524,22 @@ impl ScanProgress for SubtreeScanProgress {
             )
             .map_err(SqliteClientError::from)
         } else {
-            // Compute the number of fully scanned notes directly from the blocks table
-            let fully_scanned_size = conn.query_row(
+            let start_height = birthday_height;
+            // Compute the starting number of notes directly from the blocks table
+            let start_size = conn.query_row(
                 "SELECT MAX(sapling_commitment_tree_size)
                  FROM blocks
-                 WHERE height <= :fully_scanned_height",
-                named_params![":fully_scanned_height": u32::from(fully_scanned_height)],
+                 WHERE height <= :start_height",
```

Comment:
Nit: now that `start_height = birthday_height` there's an off-by-one here, because the birthday height is one of the blocks that gets scanned, not the base block.

Non-blocking, the error in the output is minimal, and it isn't worth re-running CI.